aml_vision/image-classification/README.md

```diff
@@ -0,0 +1,92 @@
+
+| Model                                                                                         | Max Batch Size | Accelerators | train_runtime (sec)           | train_samples_per_second      |
+|-----------------------------------------------------------------------------------------------|----------------|--------------|-------------------------------|-------------------------------|
+| [google/vit-base](https://huggingface.co/google/vit-base-patch16-224)                         | 117            | -            | 1306.508                      | 1065.129                      |
+| [google/vit-base](https://huggingface.co/google/vit-base-patch16-224)                         | 221            | ORT+DS       | 1215.846 **(~7.0% speedup)**  | 1144.553 **(~7.5% speedup)**  |
+| [apple/mobilevit-small](https://huggingface.co/apple/mobilevit-small)                         | 104            | -            | 1272.333                      | 1093.739                      |
+| [apple/mobilevit-small](https://huggingface.co/apple/mobilevit-small)                         | 124            | ORT+DS       | 1210.240 **(~5.0% speedup)**  | 1149.854 **(~5.1% speedup)**  |
+| [facebook/deit-base](https://huggingface.co/facebook/deit-base-patch16-224)                   | 117            | -            | 1287.648                      | 1080.73                       |
+| [facebook/deit-base](https://huggingface.co/facebook/deit-base-patch16-224)                   | 221            | ORT+DS       | 1235.098 **(~4.0% speedup)**  | 1126.712 **(~4.3% speedup)**  |
+| [microsoft/beit-base](https://huggingface.co/microsoft/beit-base-patch16-224-pt22k-ft22k)     | 110            | -            | 1496.612                      | 929.833                       |
+| [microsoft/beit-base](https://huggingface.co/microsoft/beit-base-patch16-224-pt22k-ft22k)     | 185            | ORT+DS       | 1338.658 **(~10.6% speedup)** | 1039.549 **(~11.8% speedup)** |
+| [miceosoft/swinv2-base](https://huggingface.co/microsoft/swinv2-base-patch4-window12-192-22k) | 69             | -            | 2653.239                      | 524.491                       |
+| [miceosoft/swinv2-base](https://huggingface.co/microsoft/swinv2-base-patch4-window12-192-22k) | 106            | ORT+DS       | 1833.086 **(~30.9% speedup)** | 759.157 **(~44.7% speedup)**  |
+
+Note: No accelerator runs use [Hugging Face's Trainer](https://huggingface.co/docs/transformers/main_classes/trainer) while ORT+DS runs use [Optimum's ORTTrainer](https://huggingface.co/docs/optimum/onnxruntime/package_reference/trainer) and pass in a deepspeed config file. For more details, inspect the difference between the `finetune-code/run_image_classification.py` and `finetune-code/run_image_classification_ort.py` files.
+
+## Run Experiments
+
+#### `image-classification/finetune-code` contains all the code that is submitted by `aml_submit.py`
```

Comment:
This sentence is confusing mostly because I see aml_submit here. Can you rephrase it? I am not adding any suggestion because I am not sure what information this sentence is conveying.